app/models/profile.server.ts

```diff
@@ -182,26 +247,6 @@ export async function consolidateProfilesByEmail(
   }
 }
 
-// interface SearchProfilesOutput {
-//   id: string
-//   email: string
-//   firstName: string
-//   lastName: string
-//   avatarUrl: string
-//   jobLevelTier: string
-//   jobLevelTitle: string
-//   department: string
-//   createdAt: string
-//   updatedAt: string
-//   country: string
-//   location: string
-//   preferredName: string
-//   benchStatus: string
-//   businessUnit: string
-//   employeeStatus: string
-//   githubUser: string
-// }
-
```

Comment:
Why did you keep this ?

Reply:
I didn't keep those, they are marked as removed 🤷 😄

Reply:
jeje it's true 😛